src/pages/api/submit_question.js

```diff
@@ -0,0 +1,41 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import { createClient, SupabaseClient } from '@supabase/supabase-js';
+
+// TODO: move the following somewhere else.
+const supabaseUrl = process.env.NEXT_PUBLIC_SUPABASE_URL || '';
+const supabaseAnonKey = process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY || '';
+const supabaseServiceKey = process.env.NEXT_PRIVATE_SUPABASE_SERVICE_KEY || '';
+const supabase = createClient(supabaseUrl, supabaseAnonKey);
+const supabaseSecret = createClient(supabaseUrl, supabaseServiceKey);
+
+export default async function handler(req, res) {
```

Comment:
```suggestion
export default async function handler(req: NextApiRequest, res: NextApiResponse) {
```
Always nice to have :)